airbyte-integrations/connectors/source-facebook-marketing/metadata.yaml

```diff
@@ -10,7 +10,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: e7778cfc-e97c-4458-9ecb-b4f2bba8946c
-  dockerImageTag: 3.0.0
+  dockerImageTag: 3.0.1
```

Comment:
nit: `3.1.0`, since this is a breaking change on the Airbyte side.

Reply:
Done